src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenInviteApplicant2Test.java

```diff
@@ -25,7 +25,7 @@
     private static final String RESPONSE = "classpath:responses/response-applicant1-invite-applicant2.json";
 
     @Test
-    public void shouldSendEmailWhenAllTemplateParamsAreValid() throws IOException {
+    public void shouldSendEmailsWhenAllTemplateParamsAreValid() throws IOException {
```

Comment:
would be nice to be more specific for e.g shouldSendEmailToApplicant1AndApplicant2